lib/common.sh

```diff
@@ -103,6 +107,8 @@ CAPM3REPO="${CAPM3REPO:-https://github.com/${CAPM3_BASE_URL}}"
 CAPIPATH="${CAPIPATH:-${M3PATH}/cluster-api}"
 CAPM3RELEASEPATH="${CAPM3RELEASEPATH:-https://api.github.com/repos/${CAPM3_BASE_URL}/releases/latest}"
 export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}")}"
+export CAPIRELEASE="${CAPIRELEASE:-v0.3.0}"
```

Comment:
you should do this the same way as it is done for CAPM3, defaulting to latest release, and being able to override to an older release. It is not a good approach to block everything with an older release (including the integration tests) just for one use-case. That use-case should be setting the environment variables to get the setup it needs. So default to latest release and override the version when doing the upgrade tests

Reply:
I decided to use v0.3.0 based on https://github.com/kubernetes-sigs/cluster-api/blob/2ac3728d26593f7c54520999477aad45934e1c59/cmd/clusterctl/hack/local-overrides.py#L51-L68.

For the local override, the version is hard coded in the code itself. I was trying to imitate that. 
But, I can do the change as you suggested as well.

Reply:
> you should do this the same way as it is done for CAPM3, defaulting to latest release, and being able to override to an older release. It is not a good approach to block everything with an older release (including the integration tests) just for one use-case. That use-case should be setting the environment variables to get the setup it needs. So default to latest release and override the version when doing the upgrade tests

Changed. 
Tested with and without ```export CAPIRELEASE=v0.3.0``
if not export, then ```v0.3.6``` is installed. v0.3.0 otherwise.